From 2fac140f228182023dc62261c3f5d626c9272614 Mon Sep 17 00:00:00 2001
From: Mike Hommey <glandium@debian.org>
Date: Sat, 8 Dec 2007 19:24:40 +0100
Subject: [PATCH] Set javascript.options.showInConsole

---
 modules/libpref/init/all.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules/libpref/init/all.js b/modules/libpref/init/all.js
index 782638b2bb..9714c8d563 100644
--- a/modules/libpref/init/all.js
+++ b/modules/libpref/init/all.js
@@ -978,6 +978,7 @@ pref("privacy.purge_trackers.consider_entity_list", false);
 pref("dom.event.contextmenu.enabled",       true);
 
 pref("javascript.enabled",                  true);
+pref("javascript.options.showInConsole",    true);
 pref("javascript.options.asmjs",                  true);
 pref("javascript.options.wasm",                   true);
 pref("javascript.options.wasm_trustedprincipals", true);
